flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/state/rocksdb/sstmerge/CompactionTracker.java

```diff
@@ -82,6 +82,10 @@ private synchronized void runIfNoManualCompactions(Runnable runnable) {
         }
     }
 
+    public synchronized void addManualCompaction() {
```

Comment:
the method name does not seem correct. `haveManualCompactions` refers to `runningManualCompactions `and `pendingManualCompactions`. I suggest `scheduleManualCompaction`.